src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,40 +509,63 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		result.append("id: ");
-		result.append(getId());
-		result.append(", lang: ");
-		result.append(getLang());
+		if (!getLang().equals("und")) {
+			result.append("Id: ").append(getId());
```

Comment:
@Sami32 Are you sure that checking if language is "und" is better than checking if id != 0? As i see it, "und" is also a setting - ```toString()``` is supposed to reveal everything relevant about the given instance. If id == 0 it means that no language is set - while "und" is also as setting as I see it. 

I also see that you removed the language name - not a bit problem for me, but some of the codes arn't that obvious (like ```iw``` for hebrew for example).